bddl/activity_definitions/put-togethera-basic-pruning-kit/problem0.bddl

```diff
@@ -4,15 +4,13 @@
     (:objects
      	pruner.n.02_1 - pruner.n.02
     	floor.n.01_1 - floor.n.01
-    	pruning_saw.n.01_1 - pruning_saw.n.01
```

Comment:
Maybe replace this with pruner.n.02?

Reply:
hm, for some reason I already have pruner.n.02